.github/workflows/build-validation.yml

```diff
@@ -9,19 +9,16 @@ on:
       - "**.cs"
       - "**.vb"
       - "**.fs"
-      - "**.cpp"
-      - "**.h"
       - "**.xaml"
       - "**.razor"
       - "**.cshtml"
       - "**.vbhtml"
-      - "**.*proj"
```

Comment:
We should keep the `csproj`, `vsproj`, and `vbproj` so that we build projects when dependencies change.

Reply:
DUH on me.